charts/clusterapi-resources/Chart.yaml

```diff
@@ -17,7 +17,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.7.1
+version: 0.8.0
```

Comment:
Oh, I have already released version 0.8.0 (and clearly forgot to bump it here), so use 0.8.1

Reply:
Ohh. Ill bump to 0.9 since it can be considered a new feature instead of a fix

Reply:
[70649bf](https://github.com/powerhome/helm-charts/pull/43/commits/70649bf2031bdded317cab8716d49a1a7b5f5fb8) should be good now